docs/concepts/executor/serve.md

```diff
@@ -129,99 +191,3 @@ The Executor you use needs to be already containerized and stored in an accessib
 ````
 
 (reload-executor)=
-## Reload Executor
```

Comment:
why is this removed?

Reply:
content duplicated here https://feat-deployment-serve--jina-docs.netlify.app/concepts/executor/hot-reload/

Reply:
removed